packages/contentful/src/crn/queries/discussions.queries.ts

```diff
@@ -52,7 +52,7 @@ export const discussionContentQueryFragment = gql`
           avatar {
             url
           }
-          teamsCollection(limit: 1) {
+          teamsCollection(limit: 2) {
```

Comment:
```suggestion
          teamsCollection(limit: 3) {
```

I believe in prod we have users with 3 teams

Reply:
I chose to keep the limit to 2 to reduce the size of the response, since in this part we only need to know if there are multiple teams assigned to this user to show "Multiple teams" next to the name in the replies section:

![image](https://github.com/user-attachments/assets/9f01f8d1-4732-4740-8719-c71f27333427)